GVFS/GVFS.UnitTests/Mock/Virtualization/Background/MockBackgroundTaskManager.cs

```diff
@@ -17,6 +17,8 @@ public MockBackgroundFileSystemTaskRunner()
 
         public List<FileSystemTask> BackgroundTasks { get; private set; }
 
+        public override bool IsEmpty => this.BackgroundTasks.Count == 0;

```

Comment:
consider using`!this.BackgroundTasks.Any()` - which I think is more idiomatic... but doesn't matter in practice

Reply:
Since it uses `List<T>`, `.Count == 0` is faster than `.Any()`.

Reply:
> Since it uses List<T>, .Count == 0 is faster than .Any().

I doubt you will find any discernable difference in this test code :)

Reply:
What made me think of this was @derrickstolee's comment on initial his confusion here. I think the code is fine either way :)

Reply:
I will leave it as is since there will be little difference and the confusion was mostly in the context of this PR and not the code in general.